Fixtures/Miscellaneous/Extensions/MySourceGenExtension/Sources/MySourceGenToolLib/library.swift

```diff
@@ -1,3 +1,11 @@
-public func GetLibraryName() -> String {
-    return "MySourceGenToolLib"
+import Foundation
+
+extension String {
+    
+    public var quotedForSourceCode: String {
```

Comment:
This is just to show the ability for the tool that's built for use by the extension to depend on its own library in turn.